res/zee-worker.js

```diff
@@ -1,3 +1,6 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
```

Comment:
this is not appropriate here as this is not a Mozilla file: `zee.js is zlib licensed, just like zlib.`

Should we have a separate file mentioning the files that we just copied over and their license terms ?